client/src/styles/_building.details.scss

```diff
@@ -8,8 +8,8 @@
   text-align: left;
 }
 
-.anticon-close {
-  color: #fff;
+.anticon-close, .building-details-title {
+  color: #fff!important;
```

Comment:
I would increase the [CSS selector specificity](https://css-tricks.com/specifics-on-css-specificity/) here, instead of using !important.